lib/desired-caps.js

```diff
@@ -0,0 +1,87 @@
+const desiredCapConstraints = {
+  platformName: {
+    presence: true,
+    isString: true,
+    inclusion: ['iOS']
```

Comment:
My understanding is that it is an Appium-level constraint, but we override it here to specify that the `platformName` can only be `iOS`.